src/v/storage/readers_cache.cc

```diff
@@ -66,13 +66,12 @@ readers_cache::put(std::unique_ptr<log_reader> reader) {
         return model::record_batch_reader(std::move(reader));
     }
 
-    if (reader->lease_range_base_offset())
```

Comment:
this doesn't look like merge skew. am I missing something? it seems to have been in the original commit introducing the readers cache: https://github.com/vectorizedio/redpanda/blob/dev/src/v/storage/readers_cache.cc#L69

Reply:
I meant the merge skew during applying a stash